pallets/manta-pay/src/lib.rs

```diff
@@ -88,7 +88,7 @@ use manta_util::{
 
 pub use crate::types::{Checkpoint, RawCheckpoint};
 pub use pallet::*;
-pub use types::{DensePullResponse, PullResponse};
+pub use types::{DensePullResponse, PullResponse, RuntimeDensePullResponse};
```

Comment:
do we need `DensePullResponse` in the pallet at all?

Reply:
We still need `DensePullResponse` which is used from client side.

Reply:
`RuntimeDensePullResponse ` will be unused if `dense_pull_ledger_diff` is going to be removed.

Reply:
we need it in the client, but not in the pallet, no?

Reply:
I added a attribute above `DensePullResponse`
```rust
#[cfg_attr(feature = "rpc", test)]
pub struct DensePullResponse
```

But it would be a minor risk(maybe it's not risk) that because this type is not from runtime, so any hacker can modify this `DensePullResponse`, and then build a new client, which still connects to our network.